composer.json

```diff
@@ -11,14 +11,14 @@
   "require": {
     "php": "^8.1.0",
     "crell/api-problem": "^3.6.1",
-    "illuminate/http": "^9.0",
-    "illuminate/support": "^9.0",
+    "illuminate/http": "^9.0 || ^10.0",
+    "illuminate/support": "^9.0 || ^10.0",
     "nyholm/psr7": "^1.5",
-    "membrane/membrane": "^0.2.0",
+    "membrane/membrane": "^0.3.0",
```

Comment:
I tried upgrading to latest membrane-core (0.6.2) but was getting errors, and 0.3.0 was the highest I could go with passing tests.

```
Time: 00:00.047, Memory: 12.00 MB

There were 2 PHPUnit errors:

1) Membrane\Laravel\ApiProblemBuilderTest::buildFromExceptionTest
The data provider specified for Membrane\Laravel\ApiProblemBuilderTest::buildFromExceptionTest is invalid
Class "Membrane\OpenAPI\Exception\CannotProcessRequest" not found

/Users/phil/src/membrane-laravel/tests/ApiProblemBuilderTest.php:98

2) Membrane\Laravel\Middleware\RequestValidationTest::catchesCannotProcessRequest
The data provider specified for Membrane\Laravel\Middleware\RequestValidationTest::catchesCannotProcessRequest is invalid
Class "Membrane\OpenAPI\Exception\CannotProcessRequest" not found

/Users/phil/src/membrane-laravel/tests/Middleware/RequestValidationTest.php:72

--

There was 1 failure:

1) Membrane\Laravel\Middleware\RequestValidationTest::registersResultInstanceInContainer
Expectation failed for method name is "instance" when invoked 1 time
Parameter 1 for invocation Illuminate\Contracts\Container\Container::instance('Membrane\Result\Result', Membrane\Result\Result Object (...)) does not match expected value.
Failed asserting that two objects are equal.
--- Expected
+++ Actual
@@ @@
         'header' => []
         'cookie' => []
         'body' => ''
+        'request' => [...]
     )
     'result' => 1
 )

/Users/phil/src/membrane-laravel/src/Middleware/RequestValidation.php:44
/Users/phil/src/membrane-laravel/tests/Middleware/RequestValidationTest.php:49

--

There was 1 risky test:

1) Membrane\Laravel\Middleware\RequestValidationTest::registersResultInstanceInContainer
This test did not perform any assertions

/Users/phil/src/membrane-laravel/tests/Middleware/RequestValidationTest.php:30

ERRORS!
Tests: 8, Assertions: 18, Errors: 2, Failures: 1, Risky: 1.
```